### PR TITLE
xfce4-taskmanager: Initial inclusion

### PR DESCRIPTION
**Pull request by EbonJaeger**
**Summary**
- Initial inclusion of xfce4-taskmanager

Ref https://github.com/getsolus/packages/issues/305

Signed-off-by: Evan Maddock <maddock.evan@vivaldi.net>

**Test Plan**

Launch the task manager and see that running applications are listed and that the monitors update.

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/x/xfce4-taskmanager/package.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+summary    : Easy to use task manager
+description: |
+    The xfce4-taskmanager component is an easy to use graphical task manager.
+builddeps  :
+    - pkgconfig(gtk+-3.0)
+    - pkgconfig(libxfce4ui-2)
+    - pkgconfig(libwnck-3.0)
+    - pkgconfig(xmu)
+setup      : |
+    %configure --sysconfdir=/usr/share
```
Comment (joebonrichie) [inline packages/x/xfce4-taskmanager/package.yml:18]: Is overriding sysconfdir necessary?

**Review by joebonrichie — COMMENTED** [review]
One Q. May as well fix builddep ordering as well

File: packages/x/xfce4-taskmanager/package.yml (hunk 1 above)
Comment (EbonJaeger) [reply packages/x/xfce4-taskmanager/package.yml:18]: I'm thinking not. It doesn't provide any config files, so it probably doesn't do anything.

**Review by joebonrichie — DISMISSED** [review]
didn't fix build ordering, but idc xD

**Comment by EbonJaeger** [comment]
The ordering is correct though, isn't it? Unless I'm failing at the alphabet

**Comment by EbonJaeger** [comment]
Nevermind, apparently I *did* fail English...

**Review by joebonrichie — APPROVED** [review]

**Pull request merged**